browser/brave_vpn/brave_vpn_service_factory.cc

```diff
@@ -59,12 +74,16 @@ KeyedService* BraveVpnServiceFactory::BuildServiceInstanceFor(
   auto shared_url_loader_factory =
       default_storage_partition->GetURLLoaderFactoryForBrowserProcess();
 
+// TODO(bsclifton) or TODO(shong):
+// see same notes above
 #if defined(OS_WIN) || defined(OS_MAC)
-  return new BraveVpnServiceDesktop(shared_url_loader_factory,
-                                    user_prefs::UserPrefs::Get(context));
-#endif
-
-#if defined(OS_ANDROID)
+  DCHECK(base::FeatureList::IsEnabled(skus::features::kSkusFeature));
```

Comment:
skus is disabled by default, but vpn is enabled by default on mac and win so isn't this going to cause a crash in dev for everyone? You need to not pass skus to BraveVpnServiceDesktop if it's disabled

Reply:
actually vpn is disabled by default, but the check is in the wrong place. `if (!brave_vpn::IsBraveVPNEnabled())` needs to move into this method

Reply:
but still you have a problem with the DCHECK because enabling vpn without also enabling skus will cause a crash in dev

Reply:
Fixed in https://github.com/brave/brave-core/commit/bc08735f11dc33351fe3845cef9a29959b4e6eb4